sycl/include/sycl/handler.hpp

```diff
@@ -2292,61 +2292,80 @@ class __SYCL_EXPORT handler {
 
   template <typename KernelName = detail::auto_name, typename KernelType,
             typename PropertiesT>
-  std::enable_if_t<
-      ext::oneapi::experimental::is_property_list<PropertiesT>::value>
-  single_task(PropertiesT Props, _KERNELFUNCPARAM(KernelFunc)) {
+  __SYCL_DEPRECATED(
+      "Use sycl::ext::oneapi::experimental::single_task (provided in the "
+      "sycl_ext_oneapi_enqueue_functions extension) instead.")
+  std::enable_if_t<ext::oneapi::experimental::is_property_list<
+      PropertiesT>::value> single_task(PropertiesT Props,
+                                       _KERNELFUNCPARAM(KernelFunc)) {
     single_task_lambda_impl<KernelName, KernelType, PropertiesT>(Props,
                                                                  KernelFunc);
   }
 
   template <typename KernelName = detail::auto_name, typename KernelType,
             typename PropertiesT>
-  std::enable_if_t<
-      ext::oneapi::experimental::is_property_list<PropertiesT>::value>
-  parallel_for(range<1> NumWorkItems, PropertiesT Props,
-               _KERNELFUNCPARAM(KernelFunc)) {
+  __SYCL_DEPRECATED(
+      "Use sycl::ext::oneapi::experimental::single_task (provided in the "
```

Comment:
This looks like a copy-paste issue:
```suggestion
      "Use sycl::ext::oneapi::experimental::parallel_for (provided in the "
```
Please fix deprecation message here and all other similar spots.

Reply:
Will fix in the other PR.

Reply:
Fixed!